src/Application/Console/Definitions/AnalyseDefinition.php

```diff
@@ -56,7 +56,7 @@ public static function get(): InputDefinition
                 null,
                 InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
                 'Format to output the result in [console, json, checkstyle]',
-                ['console']
+                config('insights.format', ['console'])
```

Comment:
I think you can't use `config()` function here. `config()` is a helper function in `Laravel`.

Reply:
Yes ! Remember PHPInsights is framework agnostic :wink: